### PR TITLE
Fix support types

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready

## Related Issues
fixes: link to the issue

## Description
Set proper support types to:
* Nozomi Networks (also set certified)
* Lokpath

Update docker image and format yml for Nozomi Networks

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/NozomiNetworks/pack_metadata.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "name": "Nozomi Networks",
     "description": "An Integration Pack of Nozomi Networks OT Security Tools",
-    "support": "Nozomi Networks",
+    "support": "parnter",
```
Comment (anara123) [inline Packs/NozomiNetworks/pack_metadata.json:4]:
```suggestion
    "support": "partner",
```

**Review by anara123 — APPROVED** [review]
See my note

**Comment by Shellyber** [comment]
@glicht can you also add to this PR `Packs/NonSupported` metadata file?
Need to merge all this packs before I can merge the validation for it. 
https://github.com/demisto/demisto-sdk/pull/925

**Comment by glicht** [comment]
Nozomi was updated here already: https://github.com/demisto/content/pull/10408. Will remove from the PR.

**Pull request merged**